### PR TITLE
Change subctl devel release to use GH CLI

### LINKED ISSUE #1413: release-0.9 subctl artifact building is broken
https://github.com/submariner-io/submariner-operator/runs/2796673627#step:6:17

### PR DESCRIPTION
**Pull request by mkolesnik**
Resolves: #1413

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: .github/workflows/release_subctl_devel.yml [hunk 1]
```diff
@@ -17,28 +17,26 @@ jobs:
...
 
       - name: Generate the subctl release artifacts
+        run: make build-cross VERSION=${{ env.SUBCTL_TAG }}
+
+      - name: Install GH CLI
         run: |
-          make build-cross VERSION=${{ env.SUBCTL_TAG }}
-          echo "BINARIES=$(find dist/ -type f -name '*.tar.xz' -printf '%p ')" >> $GITHUB_ENV
+          curl -sLo /tmp/gh.deb https://github.com/cli/cli/releases/download/v1.11.0/gh_1.11.0_linux_amd64.deb
+          sudo dpkg -i /tmp/gh.deb
```
Comment (skitt) [inline .github/workflows/release_subctl_devel.yml]:
```suggestion
          sudo apt install /tmp/gh.deb
```

to deal with dependencies if that’s ever necessary.

**Review by skitt — CHANGES_REQUESTED** [review]
A minor tweak...

**Review by mangelajo — APPROVED** [review]
Leaving my +1 for after when @skitt comments are handled.

**Comment by mkolesnik** [comment]
Make sure to quash when merging

**Review by skitt — APPROVED** [review]

**Pull request merged**